src/components/ProfilePage/QuestCarousel/QuestCarousel.tsx

```diff
@@ -1,89 +1,105 @@
-import { CarouselContainer } from '@/components/Blog/BlogCarousel/CarouselContainer';
 import { useOngoingQuests } from '@/hooks/useOngoingQuests';
-import type { Quest } from '@/types/loyaltyPass';
 import { useTranslation } from 'react-i18next';
-import { QuestCard } from '../QuestCard/QuestCard';
-import { QuestCarouselContainer } from './QuestCarousel.style';
-import { TempTitle } from './TempTitle/TempTitle';
-import { QuestCardDetailled } from '../QuestCardDetailled/QuestCardDetailled';
-import { QuestCardSkeleton } from '../QuestCardDetailled/QuestCardSkeleton';
+import { CarouselContainer } from 'src/components/Blog';
 import { checkInclusion } from 'src/components/Superfest/ActiveSuperfestMissionsCarousel/ActiveSuperfestMissionsCarousel';
+import type { OngoingRewardsItem } from 'src/hooks/useOngoingRewards';
+import { useOngoingRewards } from 'src/hooks/useOngoingRewards';
+import type { Quest } from 'src/types/loyaltyPass';
+import { QuestCardSkeleton } from '../QuestCard/QuestCardSkeleton';
+import { QuestCardDetailled } from '../QuestCardDetailled/QuestCardDetailled';
+import { QuestCarouselContainer } from './QuestCarousel.style';
 
 interface QuestCarouselProps {
-  quests?: Quest[];
-  loading: boolean;
   pastCampaigns?: string[];
 }
 
-export const QuestCarousel = ({
-  quests,
-  loading,
-  pastCampaigns,
-}: QuestCarouselProps) => {
-  const { url } = useOngoingQuests();
+export const QuestCarousel = ({ pastCampaigns }: QuestCarouselProps) => {
   const { t } = useTranslation();
-
+  const { quests, isLoading: isQuestsLoading, url } = useOngoingQuests();
+  const { data: ongoingRewards, isLoading: isOngoingRewardsLoading } =
+    useOngoingRewards();
```

Comment:
rename to `useOngoingNumericQuests` pls and `ongoingNumericQuests`

Reply:
I did rename it but I applied that name to stay consistent with backend where the endpoint is called `ongoing-rewards`.